sdk/lib/drivers/sound/mmixer/wave.c

```diff
@@ -244,13 +245,20 @@ MMixerOpenWavePin(
     LPMIXER_DATA MixerData;
     NTSTATUS Status;
     MIXER_STATUS MixerStatus;
+    DWORD cbSize;
 
     MixerData = MMixerGetDataByDeviceId(MixerList, DeviceId);
     if (!MixerData)
         return MM_STATUS_INVALID_PARAMETER;
 
+    /* Calculate additional data size basing on the format tag */
+    if (WaveFormatEx->wFormatTag == WAVE_FORMAT_PCM)
```

Comment:
Related to your previous commit: Is this `WAVE_FORMAT_PCM` only? Or `WAVE_FORMAT_EXTENSIBLE` only? As in, which case should other `WAVE_FORMAT_*` use (if that applies)?

Reply:
So other formats besides PCM and EXTENSIBLE should also use 'else' case.

Reply:
Thanks for the reference.